Core/Graphics/Include/Shared/Babylon/Graphics.h

```diff
@@ -14,9 +14,10 @@ namespace Babylon
     public:
         ~Graphics();
 
-        static std::unique_ptr<Graphics> CreateGraphics(const GraphicsConfiguration& config);
+        static std::unique_ptr<Graphics> CreateGraphics(const WindowConfiguration& config);
+        static std::unique_ptr<Graphics> CreateGraphics(const ContextConfiguration& config);
 
-        void UpdateWindow(const GraphicsConfiguration& config);
+        void UpdateWindow(const WindowConfiguration& config);
```

Comment:
I'm still unclear, can you call this function after having created a graphics object with a `ContextConfiguration`? And if so, what exactly does that do?

Reply:
native window is only used at bgfx context creation (bgfx::init). So updateWindow will do nothing but update framebuffer with backbuffer ratio.